src/lib/SDK/sdk.js

```diff
@@ -143,4 +145,18 @@ export default class SDK {
     const { data } = await this.#httpClient.get(apiUrl);
     return data;
   }
+
+  /** Get tx status
+   * @param {string} txHash hash of the transaction
+   * @param {number} tokenId ID of the token
+   * @returns {Promise<object>} Transaction information
+   */
+  async getStatus({ txHash }) {
+    if (!utils.isHexString(txHash)) {
+      throw new Error('[SDK.GetStatus] You need to pass a valid tx hash as parameter');
+    }
+
+    const signer = await this.#auth.getSigner();
```

Comment:
The auth class has a provider, so you don't need to be doing this. It's probably best to expose the provider via `provider()`

Reply:
not sure I understand , you mean i have to create a` provider()` method in the same class ?